core/templates/dev/head/app.js

```diff
@@ -71,6 +71,15 @@ oppia.constant(
   'RTE_COMPONENT_SPECS',
   window.GLOBALS.RTE_COMPONENT_SPECS ? window.GLOBALS.RTE_COMPONENT_SPECS : {});
 
+oppia.constant(
```

Comment:
Perhaps this should be constructed internally within the service, if it's only used in one place.